.storybook/preview.js

```diff
@@ -55,6 +55,99 @@ const adjustedTheme = ancestorTheme => ({
   ),
 })
 
+const globalStyles = theme => css`
```

Comment:
Can we cut it down to the bare minimum? Eg, not sure that hiding overflow to reenable it after is optimal :D

Reply:
It's currently the same CSS as before, what would you want to remove apart the duplicated `overflow`?

Reply:
I was hoping everything expect fonts 😄 

If this is a prerequisite for this ui lib to look as intended we actually should remove it ...